frontend/src/components/modal/mobile/styled-components/GridSmallItemLeft.jsx

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 export default styled.div`
-  max-height: 289px;
-  grid-column: 1 / 2;
-  cursor: pointer;
+    max-height: 289px;
+    grid-column: 1 / 2;
```

Comment:
to make your grid layout more readable, i suggest expressing your grid with `grid-template-areas` and `grid-area`

Reply:
I agree. I ended up using grid-template-areas and grid area in my other service, but here I was still learning and experimenting with grid for this one.